pipeline/db/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "db",
+    "version": "1.0.0",
+    "description": "",
+    "main": "db.js",
+    "dependencies": {
+        "pg": "^7.0.2"
+    },
+    "devDependencies": {},
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "",
+    "license": "BSD"
+}
```

Comment:
We do not want most of this interms of scripts test and devDependencies. Licence is good though. Add author too :)